scala/private/rules/scala_test.bzl

```diff
@@ -75,6 +77,8 @@ _scala_test_attrs = {
     "_lcov_merger": attr.label(
         default = Label("@bazel_tools//tools/test/CoverageOutputGenerator/java/com/google/devtools/coverageoutputgenerator:Main"),
     ),
+    "env": attr.string_dict(default = {}),
+    "inherited_environment": attr.string_list(),
```

Comment:
Could be called `env_inherit` since it should behave just like the attr of that name on e.g. `java_test`.

Reply:
Fixed, thanks!